scriptmodules/supplementary/sdl2.sh

```diff
@@ -38,6 +39,8 @@ function depends_sdl2() {
     isPlatform "mali" && depends+=(mali-fbdev)
     isPlatform "kms" && depends+=(libdrm-dev libgbm-dev)
     isPlatform "x11" && depends+=(libpulse-dev libegl1-mesa-dev libgles2-mesa-dev libglu1-mesa-dev)
+    # for vero4k start depends again due to unwanted X packages
```

Comment:
possible to avoid putting the depends again by excluding them for vero4k earlier ?

Reply:
As it's to avoid X11 packages, do you mean they could go into `isPlatform "x11"` perhaps?  Certainly works for my box.
I was basically trying not to break other peoples stuff. But if a platform doesn't bear the `x11` flag I don't see why they'd need anything X related.